nbclassic/templates/page.html

```diff
@@ -137,6 +137,26 @@
 </noscript>
 
 <div id="header" role="navigation" aria-label="{% trans %}Top Menu{% endtrans %}">
+  <div  id="newsId" style="display: none">
+    <div class="alert alert-info" role="alert">
+      <div style="display: flex">
+        <div>
+          <span class="label label-warning">NEWS</span>
+          If you plan to upgrade to Notebook 7, read <a href="https://nbclassic.readthedocs.io/en/latest/nbclassic_to_notebook7.html" target="_top">https://nbclassic.readthedocs.io/en/latest/nbclassic_to_notebook7.html</a>
```

Comment:
Thank you @echarles! Currently, the URL for the documentation is returning a 404. I'm not sure which portion of the documentation this link was intended to point to, was it maybe ["Jupyter NbClassic"](https://nbclassic.readthedocs.io/en/latest/nbclassic.html#jupyter-nbclassic) or ["NbClassic in the Jupyter Ecosystem"](https://nbclassic.readthedocs.io/en/latest/)?

Reply:
Thx for the review @RRosio. My last commit creates a dedicated documentation page `nbclassic_notebook` which aims to gather explanations around compatibility/... of NbClassic with the various versions of Notebook. The banner points to that page (returns a 404 for now as the RDT is not deployed).

That commit in the jupyterlab repo would need to be reviewed to point to that new page https://github.com/jupyterlab/jupyterlab/pull/13830/files#diff-1e189eeca647852c8b1d93c85472cc5e1a55a28ab1e564099b9f6c4292487ea8R95

WDYT?

Reply:
Thank you for clarifying, that sounds great! I'll keep an eye on this pr so I can help any updates like the one you mentioned!